src/components/MoneyCreditModal.vue

```diff
@@ -46,9 +46,18 @@
             "
             class="amount custom-montant-input"
           >
-            <h2 class="frame3-sub-title mt-3 mb-3">
+            <h2 class="frame3-sub-title mt-3">
               {{ $gettext("Top up amount") }}
             </h2>
+            <div class="ml-2 min-credit-amount">
+              {{
+                min_credit_amount[selectedCreditAccount?.backend] !== undefined
+                  ? $gettext("Minimum credit amount: ") +
+                    `${min_credit_amount[selectedCreditAccount?.backend]}
+                    ${selectedCreditAccount?.curr}`
```

Comment:
Should probably not display the message if the limit is 0

Reply:
i made all the requested changes. thank you for the review !